### PR TITLE
Adds an options parameter to "loadStylesheet"

### PR DESCRIPTION
**Pull request by harm-less**
Hey,

Guess what, I'm pretty much ready with a first release. A few minutes ago I published the repo and made sure most is working. I think everything is working out better than I expected as the hooks in UI Router 1.0 are awesome :D

The only things left are some failing units tests in TravisCI and a feature I build into this repo (hence the PR).

Please review it and let me know what you think from this PR and the new repo you can find here:
https://github.com/harm-less/angular-ui-router-css

Harm

### CODE REVIEW

**Comment by slavafomin** [comment]
Hello Harm!

I've seen your demo and it works great, good job!

I will study it further and will response to this PR soon.

File: README.md [hunk 1]
```diff
@@ -37,18 +37,25 @@ You should use minified version (`betsol-load-stylesheet.min.js`) in production.
...
 
 The function will return a reference to the `<link>` DOM element
 for your convenience. You can track added links that way an remove them
 if necessary in the future.
 
+### Options
+
+The third parameter is an optional options object. The following options are supported:
+
+```insertBefore```: This should be another element which is part of your HTML's head
```
Comment (slavafomin) [inline README.md]: Why are you using three commas instead of just one?

**Review by slavafomin — CHANGES_REQUESTED** [review]
The PR looks great, thanks! I've added some very trivial comment, please address it and I will happily merge.

File: README.md (hunk 1 above)
Comment (harm-less) [reply README.md]: It's the way I always used it, I didn't know one resulted in the same thing. It seems just one does the trick too when you don't have to specify the language. Thanks for pointing that out.

**Comment by harm-less** [comment]
I fixed it, this meets your requirements, right?

**Comment by slavafomin** [comment]
Great, thanks! Just merged it. Going to make a release soon.

**Comment by slavafomin** [comment]
I've just released version `1.1.0`, feel free to check it out. And thanks again! Cheers ;)

**Pull request merged**